cms/templates/js/course-outline.underscore

```diff
@@ -78,8 +78,14 @@ if (staffOnlyMessage) {
 }
 
 var gradingType = gettext('Ungraded');
+var gradingPolicyMismatch = false;
 if (xblockInfo.get('graded')) {
     gradingType = xblockInfo.get('format')
+    if (gradingType) {
+        gradingPolicyMismatch = (
+            xblockInfo.get('course_graders').filter((cg) => cg.toLowerCase().includes(gradingType.toLowerCase()))
```

Comment:
I think I initially was implementing it with a slightly different approach and made some assumptions about the format of the names passed in, then eventually found this much simpler approach but forgot to update this bit of implementation since it worked fine. This suggestion works great and is much cleaner, thanks!

However on a related note, I've noticed when testing out, that the "Assignment Type Name" field in the grading settings doesn't have any validation on it, so you could technically submit a name with extra trailing spaces or even a blank name, in that case this validation would only work for exact matches. This is probably a separate issue, that we'd might need to address in another ticket if this is not intended behavior.

Reply:
That sounds like a totally separate issue, so I don't think we need to worry about it for now. Good catch though!